fuel-vm/src/checked_transaction.rs

```diff
@@ -153,7 +155,7 @@ impl<Tx: IntoChecked> Borrow<Tx> for Checked<Tx> {
 /// Performs checks for a transaction
 pub trait IntoChecked: FormatValidityChecks + Sized {
     /// Metadata produced during the check.
-    type Metadata: Sized;
+    type CheckedMetadata: Sized + Clone;
```

Comment:
this require clone now?

Reply:
yes, otherwise we can't clone the checked transaction

Reply:
Ok so then we can remove this clone?